### DISCUSSION TITLE
N64 Cart Flashram Issue

### DISCUSSION THREAD

**Discussion by majorpbx** [sanni/cartreader#245] [Need help]
On some games for N64 I get the following when trying to dump Flashram. ROM dumps perfectly without any issue. This may be a known issue but thought I'd mention it here.

Game: Pokemon Stadium
ID: CPSJ

Chip Info,

Make: SHARP
Type: LH52V246
Stamp: 9828 3 D

> 0, 1, 8A, 93, 8B, 50, 0, 0, Flashram unknown

**Comment by sanni** [comment]
That's an sram chip not flashram.
Use the force save type option to change save type to sram.

![pmon-sram1](https://user-images.githubusercontent.com/19169494/132945958-df52ef99-284b-4ffe-ac95-66d760bcafa3.JPG)

**Reply by majorpbx** [reply]
Well knew it would be a simple answer, does this mean the n64.txt file needs to be updated?

**Reply by sanni** [reply]
Yes I have updated the N64.txt too. 😃
